internal/s3/s3.go

```diff
@@ -394,13 +411,20 @@ func (s *s3Storage) Fetch(ctx context.Context, name string, unpack bool, output
 				return warns, errCtx.Wrap(errGo).With("fileType", fileType).With("stack", stack.Trace().TrimRuntime())
 			}
 
-			path, _ := filepath.Abs(filepath.Join(output, header.Name))
-			if !strings.HasPrefix(path, output) {
-				fmt.Println(errCtx.NewError("archive file name escaped").With("path", path, "output", output, "filename", header.Name).With("stack", stack.Trace().TrimRuntime()).Error())
+			if willEscape(header.Name, output) || willEscape(header.Name, output) {
 				return warns, errCtx.NewError("archive file name escaped").With("filename", header.Name).With("stack", stack.Trace().TrimRuntime())
```

Comment:
Why do we do this twice? willEscape() is just a function with no side-effects or not?

Reply:
The function has no side effects.  It is called twice because the symbolic link has two entries implicated, one is the target of the link and the other is the location of the link itself.

Reply:
Yeah, OK. But we have a function with no side effects called twice with identical parameters.

Reply:
Ahh yes a bug.  I need the symbolic link target in there.